client/utils/help-category-map.json

```diff
@@ -125,5 +125,6 @@
   "learn-parent-graphs-and-polynomials": "Python",
   "learn-quadratic-equations": "Python",
   "learn-simple-and-compound-interest": "Python",
-  "college-algebra-with-python-conclusion": "Python"
+  "college-algebra-with-python-conclusion": "Python",
+  "learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard": "HTML-CSS"
```

Comment:
This should point to the JavaScript curriculum

Reply:
done